test/unit/mdc-textfield/foundation.test.js

```diff
@@ -772,14 +772,16 @@ test('mousedown on the input sets the line ripple origin', () => {
 
 test('touchstart on the input sets the line ripple origin', () => {
   const {foundation, mockAdapter} = setupTest();
-  const mockEvt = {
-    target: {
-      getBoundingClientRect: () => {
-        return {};
+  const mockTouchStartEvent = {
```

Comment:
Should we update the `verify` call to expect a specific value passed to `setLineRippleTransformOrigin` in order to better test this fix?

Reply:
Done.